nuget/AppCenterCrashes.nuspec

```diff
@@ -38,6 +43,7 @@
     <!-- UWP -->
     <file src="$uwp_dir$/Microsoft.AppCenter.Crashes.dll" target="lib/uap10.0" />
     <file src="$uwp_dir$/Microsoft.AppCenter.Crashes.xml" target="lib/uap10.0" />
+    <file src="SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.UWP/Properties/Microsoft.AppCenter.Crashes.UWP.rd.xml" target="lib/uap10.0/Microsoft.AppCenter.Crashes/Properties/Microsoft.AppCenter.Crashes.rd.xml" />
```

Comment:
Remove anything related to the RD file and then we can merge that to the feature branch. Just confirmed it does not solve the exception we are seeing, and the exception is actually recovered (if no debugger attached it would not break into exceptions and everything works, no need to bump sdk version, no need for rd.xml file).